processor_function/shared/cnab_builder.py

```diff
@@ -29,22 +29,41 @@ def __init__(self, resource_request_message: dict):
         self._container_group_name = ""
         self._location = ""
 
-    def _build_porter_cmd_line(self) -> str:
+    def _build_porter_cmd_line(self) -> List[str]:
```

Comment:
Command line is an interface. I'd rename this "_build_porter_command".

Reply:
Soooooo many magic strings here.

Reply:
changed